docs/configuration.rst

```diff
@@ -84,6 +85,52 @@ Note that, equivalently, you can add these during webapp configuration:
             config.include('pyramid_swagger')
 
 
+.. _user-format-label:
+
+user_formats (Swagger 2.0 only)
+---------------------------------------
+
+The option ``user_formats`` provides user defined formats which can be used
+for validations/format-conversions. This options can only be used via webapp
+configuration.
+
+Sample usage:
+
+.. code-block:: python
+
+        def main(global_config, **settings):
+            # ...
+            settings['pyramid_swagger.user_formats'] = [user_format]
+
+
+``user_format`` used above is an instance of
+:class:`bravado_core.formatter.SwaggerFormat` (`ref <http://bravado-core.readthedocs.org/en/latest/bravado_core.html#bravado_core.formatter.SwaggerFormat>`_) and can be defined like this:
+
+.. code-block:: python
+
+        import base64
+        from pyramid_swagger.tween import SwaggerFormat
+        user_format = SwaggerFormat(format='base64',
+                                    to_wire=base64.b64encode,
```

Comment:
How should pyramid_swagger users expect the to_python and to_wire arguments to be used in practice? Are they required or is the fallback behavior defined? How do they interact with the default SerDe behavior of the primitive type we're describing?